main.py

```diff
@@ -73,30 +71,24 @@ def run_many(times) -> None:
 
 
 def shutdown() -> NoReturn:
-
-    if "redditid" in globals() :
+    if "redditid" in globals():
         print_markdown("## Clearing temp files")
         cleanup(redditid)
-        print("Exiting...")
-        sys.exit()
     
     print("Exiting...")
     sys.exit()
 
-        
-
 
 if __name__ == "__main__":
     if sys.version_info.major != 3 or sys.version_info.minor != 10:
         print("Hey! Congratulations, you've made it so far (which is pretty rare with no Python 3.10). Unfortunately, this program only works on Python 3.10. Please install Python 3.10 and try again.")
         sys.exit()
-    ffmpeg_install() # install ffmpeg if not installed
+    ffmpeg_install()
     directory = Path().absolute()
     config = settings.check_toml(
         f"{directory}/utils/.config.template.toml", f"{directory}/config.toml"
     )
-    if not config :
-        sys.exit()
+    config is False and sys.exit()
         
     if (
         not settings.config["settings"]["tts"]["tiktok_sessionid"]
```

Comment:
where is my change ?

Reply:
This line is efficient enough, I don't know why you changed it

Reply:
@OpenSourceSimon